cmd/tr31cli/doc.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/moov-io/tr31"
+)
+
+func help() {
+	fmt.Printf(strings.TrimSpace(`
+tr31cli is a Go library implementing the TR-31 (ANSI X9.143) key block standard for secure cryptographic key exchange.
+
+USAGE
+   tr31cli [-v] [-algorithm] [-ik] [-tk] [-ep] [-dp] [-gm] [-en] [-de]
```

Comment:
We should call this tool `tr31` and drop the "cli" suffix.

Reply:
fixed with new pull request